CCP_App/src/main/java/com/example/demo2/exception/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,106 @@
+/************************************************************************
+ *                                                                      *
+ *  DDDD     SSSS    AAA        Daten- und Systemtechnik Aachen GmbH    *
+ *  D   D   SS      A   A       Pascalstrasse 28                        *
+ *  D   D    SSS    AAAAA       52076 Aachen-Oberforstbach, Germany     *
+ *  D   D      SS   A   A       Telefon: +49 (0)2408 / 9492-0           *
+ *  DDDD    SSSS    A   A       Telefax: +49 (0)2408 / 9492-92          *
+ *                                                                      *
+ *                                                                      *
+ *  (c) Copyright by DSA - all rights reserved                          *
+ *                                                                      *
+ ************************************************************************
+ *
+ * Author : Riyaardh Adam
+ * Created: 2023/02/21 10:27
+ *
+ ***********************************************************************/
+package com.example.demo2.exception.handler;
+
+import com.example.demo2.custom.exception.NoElementFountException;
+import com.example.demo2.custom.exception.NonUpdateFieldException;
+import com.example.demo2.custom.exception.RequiredInputNotSuppliedException;
+import com.example.demo2.viewmodel.ErrorModel;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import javax.validation.ConstraintViolationException;
+import java.util.Date;
+import java.util.Locale;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+	/**
+	 * Global exception handler that will be executed when an exception of type "Exception" is thrown.
+	 * @param exception the exception that has been thrown with the exception information.
+	 * @return a error model and a http response code of 500.
+	 */
+	@ExceptionHandler
+	public ResponseEntity<?> handleException(Exception exception){
```

Comment:
Normally with our "fallback" exception we put it last since it is the least helpful